webpack.config.js

```diff
@@ -1,116 +0,0 @@
-var webpack = require('webpack');
```

Comment:
This webpack config was used to generate the UMD build as well as bundle up CSS.
We no longer need it because rollup takes care of this.